lib/authlogic/session/base.rb

```diff
@@ -1825,21 +1857,30 @@ def params_credentials
         controller.params[params_key]
       end
 
+      def headers_credentials
```

Comment:
Should we make the following explicit?

```suggestion
      def headers_credentials
        # Setting headers_key to nil is the accepted way to disable
        # single_access_token in headers.
        return nil if headers_key.nil?
```

Or, should we just add a comment?